src/app/ubs-admin/components/ubs-admin-certificate/ubs-admin-certificate.component.scss

```diff
@@ -0,0 +1,293 @@
+
+.column-title {
+  color: var(--ubs-secondary-light-grey);
+  font-size: 14px;
+}
+
+.amount-items {
+  width: 100px;
+  height: 36px;
+  margin-right: 39px;
+  border: 1px solid var(--ubs-primary-light-grey);
+  background-color: var(--primary-white) !important;
+  border-radius: 4px;
+  font-size: 14px;
+  color: var(--ubs-secondary-grey);
+}
+
+.filter-button .mat-icon {
+  font-size: 12px;
+}
+
+.view-table-btn .mat-icon {
+  padding-top: 5px;
+  font-size: 15px;
+}
+
+.export-excel-btn {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 290px;
+  height: 36px;
+  background-color: var(--primary-white) !important;
+  border: 1px solid var(--ubs-quaternary-dark-grey) !important;
+  color: var(--ubs-quaternary-dark-grey);
+  border-radius: 4px;
+  font-weight: 700;
+  font-size: 14px;
+  font-family: var(--primary-font);
+  transition: 0.2s linear;
+
+  &:hover {
+    color: var(--ubs-secondary-grey);
+    border: 1px solid var(--ubs-secondary-grey) !important;
+  }
+}
+
+.view-table-btn {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 154px;
+  height: 36px;
+  background-color: var(--primary-white) !important;
+  border: 1px solid var(--ubs-quaternary-dark-grey) !important;
+  color: var(--ubs-quaternary-dark-grey);
+  border-radius: 4px;
+  font-weight: 700;
+  font-size: 14px;
+  font-family: var(--primary-font);
+  margin-right: 20px;
+  margin-left: auto;
+  transition: 0.2s linear;
+
+  &:hover {
+    color: var(--ubs-secondary-grey);
+    border: 1px solid var(--ubs-secondary-grey) !important;
+  }
+}
+
+.research {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 426px;
+  height: 36px;
+  border: 1px solid var(--ubs-primary-light-grey);
+  border-radius: 4px;
+  margin-top: 20px;
+  margin-bottom: 32px;
+}
+
+.select-group {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+
+  img {
+    position: absolute;
+    top: 50%;
+    left: 0;
+    transform: translate(24px, -50%);
+    pointer-events: none;
+  }
+}
+
+.display-counter {
+  padding: 8px 24px 8px 48px;
+  border-radius: 3px;
+  border: 1px solid var(--ubs-secondary-grey);
+  appearance: none;
+  font-size: 14px;
+  color: var(--primary-dark-grey);
+  outline: none;
+}
+
+.search-input {
+  width: 100%;
+}
+
+.search-input input {
+  padding: 0 0 0 15px;
+  margin: 0;
+  width: 100%;
+  background: transparent;
+  border: none;
+  outline: none;
+}
+
+.icon-style {
+  color: var(--ubs-primary-light-grey);
+  margin: 8px;
+}
+
+.dropdown-menu input {
+  width: 16px;
+  z-index: 9;
+  border-radius: 3px;
+  border: 1px solid var(--ubs-secondary-grey);
+}
+
+.mat-header-cell .mat-icon {
+  padding-top: 5px;
+  padding-left: 10px;
+  font-size: 12px;
+
+  &:hover {
+    cursor: pointer;
+  }
+}
+
+.dropdown-menu {
+  height: 480px;
+  min-width: 356px;
+  margin-right: 5px;
+  overflow: scroll;
+  padding: 24px;
+  margin-top: 170px;
```

Comment:
Combine this into one:
margin: 170px 5px 0 0

Reply:
done